Crypto/Tests/KeyPairTests.swift

```diff
@@ -68,5 +71,28 @@ class KeyPairTests: XCTestCase {
             try! KeyPair.destroy(tag: tag)
         }
     }
+
+    func testStoreKeyPairSuccessfully() throws {
+        let keyPair: KeyPair = try bundle.decodable(fromJSON: "asymDonationKey")
+        let tag = UUID().uuidString
+        let algo = RSAAlgorithm()
+        try keyPair.store(tag: tag)
+        let fetchedKeyPair = try KeyPair.load(tag: tag, algorithm: algo)
+        XCTAssertEqual(try keyPair.publicKey.asBase64EncodedString(),
+                       try fetchedKeyPair.publicKey.asBase64EncodedString())
+        XCTAssertEqual(try keyPair.privateKey.asBase64EncodedString(),
+                       try fetchedKeyPair.privateKey.asBase64EncodedString())
+        try KeyPair.destroy(tag: tag)
```

Comment:
Could you add a new line between the different phases and add the Phase names as comments (you this Given-When-Then thing). It makes stuff easier to spot.